screens/product-hor.html

```diff
@@ -0,0 +1,431 @@
+                                        <input type="number" value="1" class="item-qty text-center">
+                                        <button class="btn btn-link"><i class="fa fa-solid fa-minus"></i></button>
+                                    </div>
+                                    <div class="children-stacked grid-gap">
+                                        <button class="fa fa-shopping-cart btn btn-secondary">Add to cart</button>
+                                        <button class="fa fa-solid fa-heart btn btn-secondary">Wishlist</button>
+                                    </div>
+                                </div>
+
+                            </div>
+
+                        </li>
+
+                        <li>
+                            <div class="card card-hor d-flex">
+                                <img class="card-media"
+                                    src="https://i0.wp.com/stationerie.co.in/wp-content/uploads/2021/05/1A3B627D-C304-4070-992B-CA499237F962-scaled.jpeg?resize=600%2C600&ssl=1"
+                                    alt="" />
+                                <div class="card-data children-stacked p-rel">
+                                    <div class="card-title">BRUSTRO Artists ’ White Bristle Set </div>
+                                    <span class="product-rating"> <i class="fa fa-solid fa-star"></i> 3.5</span>
+                                    <div class="d-flex grid-gap">
+                                        <span class="curr-price text-green">
+                                            ₹697
+                                        </span>
+                                        <span class="prev-price text-linethrough">
+                                            ₹349
+                                        </span>
+                                    </div>
+
+                                    <div class="btn-counter d-flex">
+                                        <button class="btn btn-link"><i class="fa fa-solid fa-plus"></i></button>
+                                        <input type="number" value="1" class="item-qty text-center">
+                                        <button class="btn btn-link"><i class="fa fa-solid fa-minus"></i></button>
+                                    </div>
+                                    <div class="children-stacked grid-gap">
+                                        <button class="fa fa-shopping-cart btn btn-secondary">Add to cart</button>
+                                        <button class="fa fa-solid fa-heart btn btn-secondary">Wishlist</button>
+                                    </div>
+                                </div>
+
+                            </div>
+
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+
+    </div>
+</body>
+
+</html>
+
+
+
+
+
+<!-- <html lang="en">
```

Comment:
If not needed then remove the commented part.

Reply:
Thanks for this! I had forgotten, removed it now.